webapp/channels/src/components/youtube_video/youtube_video.tsx

```diff
@@ -75,6 +76,11 @@ export default class YoutubeVideo extends React.PureComponent<Props, State> {
                     height='360px'
                     frameBorder='0'
                     allowFullScreen={true}
+                    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
```

Comment:
Do we need to allow all these things? `picture-in-picture` and `encrypted-media` are the only ones that seem like they would be applicable to a video in MM. `autoplay` is technically applicable, but I don't think we want autoplaying video in MM

Reply:
It's necessary and the feature has been on `src` as `autoplay=1`.
| Without autoplay | With autoplay |
| --- | --- |
| <video src="https://github.com/mattermost/mattermost/assets/26380261/f7d1ea07-7a62-47d9-867d-23851b2232d1"> | <video src="https://github.com/mattermost/mattermost/assets/26380261/9d3428b8-eee8-4ad3-99bb-a32e51dded73"> |

Reply:
Oh right. I forgot we went from showing a static image to the actual video. That makes sense.

I guess web-share/clipboard-write are needed for a share button and accelerometer/gyroscope are needed for rotating the video then